.github/CODEOWNERS

```diff
@@ -1516,12 +1516,38 @@ x-pack/plugins/cloud_integrations/cloud_full_story/server/config.ts @elastic/kib
 #CC# /x-pack/plugins/security/ @elastic/kibana-security
 
 # Response Ops team
-/x-pack/test/examples/triggers_actions_ui_examples @elastic/response-ops
+/x-pack/test/functional/page_objects/maintenance_windows_page.ts @elastic/response-ops
+/x-pack/test_serverless/functional/test_suites/observability/screenshot_creation/index.ts @elastic/response-ops
+/x-pack/test_serverless/functional/test_suites/observability/rules/rules_list.ts @elastic/response-ops
+/x-pack/test/functional_with_es_ssl/config.base.ts @elastic/response-ops # Assigned per https://github.com/elastic/kibana/pull/197070
+/x-pack/test/functional_with_es_ssl/lib/alert_api_actions.ts @elastic/response-ops
+/x-pack/test/functional_with_es_ssl/lib/get_test_data.ts @elastic/response-ops
+/x-pack/test/functional_with_es_ssl/page_objects/index.ts @elastic/response-ops # Assigned per git blame
+/x-pack/test/functional_with_es_ssl/page_objects/triggers_actions_ui_page.ts @elastic/response-ops
+/x-pack/test/functional_with_es_ssl/page_objects/rule_details.ts @elastic/response-ops
+/x-pack/test/functional_with_es_ssl/lib/object_remover.ts @elastic/response-ops
+/x-pack/test/stack_functional_integration/apps/alerts @elastic/response-ops
+/x-pack/test/functional/services/actions @elastic/response-ops
+/x-pack/test/api_integration_basic/apis/security_solution/index.ts @elastic/response-ops
+/x-pack/test/api_integration_basic/apis/security_solution/cases_privileges.ts @elastic/response-ops
+/x-pack/test/upgrade/services/rules_upgrade_services.ts @elastic/response-ops
+/x-pack/test/upgrade/apps/rules @elastic/response-ops
+/x-pack/test/examples/triggers_actions_ui_examples @elastic/response-ops # Assigned per https://github.com/elastic/kibana/blob/main/x-pack/examples/triggers_actions_ui_example/kibana.jsonc#L4
+/x-pack/test/functional/services/rules @elastic/response-ops
+/x-pack/test/plugin_api_integration/plugins/sample_task_plugin @elastic/response-ops
+/x-pack/test/functional/es_archives/rule_exceptions @elastic/response-ops
```

Comment:
this is security

Reply:
https://github.com/elastic/kibana/pull/199797/commits/ebf1ede6aa44e5b2486dd8d996e1b1eb01532f05